index.html

```diff
@@ -0,0 +1,62 @@
+            alt="the full microverse logo with a happy female developer in the background"
+            width="20%"
+            height="20%"
+        /></a>
```

Comment:
I think you could do a better job with your space usage and indentation here. Make your code look easier on the eye. Go to a new line only when it is necessary.